molecule/default/testvars.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Is it common to include the default role variables into these tests? Or did you just include them to allow for easier modification in the future

Reply:
This is a way of testing having a general scenario without using the defaults, in case a different value is needed, I do like having that testvars file to have that separation and flexibility.

Reply:
Okay I see :)